src/api/adminViewApiSlice.ts

```diff
@@ -0,0 +1,43 @@
+import {apiSlice} from "./apiSlice";
+import { ChangePasswordRequest } from 'types';
+
+interface HrRegisterData {
+    fullName: string;
+    company: string;
+    email: string;
+    maxReservedStudents: number;
+
+}
+
+export const adminViewApiSlice = apiSlice.injectEndpoints({
+    endpoints: (builder) => ({
+        sendHr: builder.mutation({
+            query: (hrRegisterData: HrRegisterData) => ({
```

Comment:
sendHr: build.mutation<ToCoZwracaResponse, toCoBędzieszPodawałDoFunkcjiZHooka>(
Na przyszłość aby zachować typy dla Typescriptu możesz kożystać z typów generycznych. Pierwszy to typ opisujący co zwraca response a drugi to typ tego co wrzucasz do funkcji którą otrzymujesz z hooka. Dzięki temu w programie mamy zachowane prawidłowo typy. Teraz dla typescriptu poprostu mamy any